### ISSUE TITLE
Instuctions, please

### ISSUE THREAD

**Issue by NotepadPlusUser** [mi-g/vdhcoapp#85]
Thank you for the addon, and for continuing to update VdhCoApp.  It is a very useful addon, and very easy to use.

I am not a techie, and I have forgotten now how to deal with the file 
    VdhCoAppSetup-1.6.0.exe
that I have just downloaded.

* Do I run it the same as any other installer file? 
* Are there different instructions for Vivaldi and Firefox? 
* Are there licensing issues?
* Is it fhe EXE file that is intended for Windows 10? 
* Is the installer the same for 32-bit and 64-bit? 
* In straightforward non-techie language, what are the changes?

Brief instructions about these things would be appreciated at the top of your GitHub download webpage 
    https://github.com/mi-g/vdhcoapp/releases
.

**Comment by mi-g** [comment]
It works like most Windows application installs: there is an installer file `VdhCoAppSetup-1.6.0.exe`. You execute it and you will be guided.
    
- Do I run it the same as any other installer file? **yes**
- Are there different instructions for Vivaldi and Firefox? **no** but Vivaldi is not supported, at leat officially
- Are there licensing issues? **no** it's pure GPL
- Is it fhe EXE file that is intended for Windows 10? **yes** and for Windows 7 too
- Is the installer the same for 32-bit and 64-bit? **yes** the installer contains both architectures and will install the good one automatically
- In straightforward non-techie language, what are the changes? **mostly cleanup** plus additional packages for older linux flavours

**Comment by NotepadPlusUser** [comment]
Dear Michel,

Thanks very much for your careful reply.  My post was intended as a constructive suggestion that could help users not experienced in computing.  I knew that the EXE file was the file needed for Windows, and I guessed that the other things would become clear when I ran the installer --- although because it was for a browser extension, I was a little nervous running it without reference to any browser.

I hope that my post didn't come across too negative, because I have used your addon many, many times very successfully, and I am certainly very pleased that I have purchased it.

All the best for your work.  

Bill Pender, Sydney, Australia

Friday, December 18, 2020, 8:38:30 PM, you wrote:


It works like most Windows application installs: there is an installer file VdhCoAppSetup-1.6.0.exe. You execute it and you will be guided.
Do I run it the same as any other installer file? yes
Are there different instructions for Vivaldi and Firefox? no but Vivaldi is not supported, at leat officially
Are there licensing issues? no it's pure GPL
Is it fhe EXE file that is intended for Windows 10? yes and for Windows 7 too
Is the installer the same for 32-bit and 64-bit? yes the installer contains both architectures and will install the good one automatically
In straightforward non-techie language, what are the changes? mostly cleanup plus additional packages for older linux flavours
